packages/imperative/__tests__/__integration__/cmd/__tests__/integration/cli/auth/Cmd.cli.auth.login.fruit.integration.test.ts

```diff
@@ -62,7 +62,6 @@ describe("imperative-test-cli auth login", () => {
         expect(response.stdout.toString()).toContain("certFile:    " + fakeCertPath);
         expect(response.stdout.toString()).toContain("certKeyFile: " + fakeCertKeyPath);
         expect(response.stdout.toString()).toContain("tokenType:   jwtToken");
-        expect(response.stdout.toString()).toContain("tokenValue:  (secure value)");
```

Comment:
Curious why this was removed?

Reply:
that line had a spacing issue. adding it back in with fixed spacing